force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +487,12 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+        if (person1.age > person2.age){
+            return person1;
+        } else if (person2.age > person1.age){
+            return person2;
+        } else {
+            return person1;
+        }
```

Comment:
Here we can shorten things up by using `>=` in the first if statement:
```suggestion
        if (person1.age >= person2.age){
            return person1;
        } else {
            return person2;

```
And when you learn or feel more comfortable with the **ternary operator**, you can shorten it to one line of code:
`return person1 > = person2 ? person1 : person2;`